data.sql

```diff
@@ -21,4 +21,15 @@ VALUES (9, 'Boarmon', '2005-06-07',7, true,20.4);
 INSERT INTO animals (ID,NAME,date_of_birth,escape_attempts,neutered,weight_kg) 
 VALUES (10, 'Blossom', '1998-10-13',3, true,17);
 INSERT INTO animals (ID,NAME,date_of_birth,escape_attempts,neutered,weight_kg) 
-VALUES (11, 'Ditto', '2022-05-14',4, true,22)
\ No newline at end of file
+VALUES (11, 'Ditto', '2022-05-14',4, true,22);
+
+-- insertion to owners table
+INSERT INTO owners(full_name,age) VALUES('Sam Smith',34);
+INSERT INTO owners(full_name,age) VALUES('Jennifer Owl',19);
+INSERT INTO owners(full_name,age) VALUES('Bob',45);
+INSERT INTO owners(full_name,age) VALUES('Melody Pond',77);
+VALUES VALUES owners(full_name,age) VALUES('Dean Whinchester',14);
+VALUES INTO owners(full_name,age) VALUES('Jodie Whittaker',38);
+-- INSERTION TO SPECIES TABLE
+INSERT INTO species(name) values ('Pokemon');
+INSERT INTO species(name) VALUES ('Digimon');
```

Comment:
- [ ] Kindly implement the following requirements in the screenshots in `data.sql` as they are missing

#### Modify your inserted animals so it includes the species_id value:
- If the name ends in "mon" it will be Digimon
- All other animals are Pokemon

![image](https://user-images.githubusercontent.com/81932478/213406547-b5ae8455-74d9-494a-8d79-5bf9666c7504.png)

#### Modify your inserted animals to include owner information (owner_id):
- Sam Smith owns Agumon.
- Jennifer Orwell owns Gabumon and Pikachu.
- Bob owns Devimon and Plantmon.
- Melody Pond owns Charmander, Squirtle, and Blossom.
- Dean Winchester owns Angemon and Boarmon.

![image](https://user-images.githubusercontent.com/81932478/213407118-c24743db-ff2b-4059-ac95-c33af1be3610.png)